origin-discovery/src/apollo/db.js

```diff
@@ -30,6 +30,15 @@ async function getListings (listingIds) {
   // Note: preserve ranking by keeping returned listings in same order as listingIds.
   const listings = []
   listingIds.forEach(id => {
+    let display = "normal"
+    /* hidden listings are not returned right now, but at some point in the future
```

Comment:
super nit: "returned" is a bit confusing. clarify that no hidden listing ids are passed to this method (since they get filtered at the layer above).

Reply:
Good catch, I have changed the comment.